packages/app/src/components/nominationInfo/healthProviderSearch/styles.css

```diff
@@ -0,0 +1,56 @@
+.new-files-table {
```

Comment:
As discussed, let's see if we can make the existing css reusable instead of creating this new copy.